### DISCUSSION TITLE
Is it possible to disable/enable saving clips for individual zones?

### DISCUSSION THREAD

**Discussion by hawkeye217** [blakeblackshear/frigate#425] [Ask A Question] [answered]
I have a camera on my property that is tracking both ``person`` and ``car``, and I have a couple of zones set up on it: ``lawn`` and ``street``.  

I use the ``street`` zone and a template sensor in Home Assistant to track the number of cars that pass my house every day.  I don't need to save clips for the ``street`` zone, but I do want to save clips for the ``lawn`` zone.  

Is this currently possible?

**Comment by blakeblackshear** [comment, marked as answer]
It's not.
